sound/soc/sof/control.c

```diff
@@ -112,6 +139,9 @@ int snd_sof_switch_put(struct snd_kcontrol *kcontrol,
 		cdata->chanv[i].channel = i;
 	}
 
+	if (scontrol->led_ctl.use_leds)
+		update_acpi_mute_led(scontrol, kcontrol, ucontrol);
```

Comment:
is the reference to ACPI needed here, since it's abstracted away by the ledtrig library?

Reply:
yes will change

Reply:
done